README.md

```diff
@@ -1 +1,58 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> This "Hello-world" project purpose is to practice the GitHub work flow.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://github.com/rbreva/Hello-world.git)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+No prerequisite required
+
+## Authors
+
+👤 **R Mauricio Mendoza H**
+
+- GitHub: [GitHub](https://github.com/rbreva)
+- Behance: [Behance](https://www.behance.net/rbreva)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/r-mauricio-mendoza-huerta-0782a9166/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

![00Mit](https://user-images.githubusercontent.com/79058364/182822618-fdfb2523-cb23-4060-9f72-75ec8c29e4cd.png)


#### The reason for this is that there is no `MIT.md` file in the root directory of this project. Kindly attach the file. 👍